### ISSUE TITLE
WCAG 4.1.2: Ensures buttons have discernible text (button)

### ISSUE THREAD

**Issue by apfeldieb** [microsoft/accessibility-insights-windows#1301]
#### Issue
`Ensures buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[(1) Microsoft Edge | State of the Platform - YouTube](https://www.youtube.com/watch?v=sU0WRZ0kkNo)
#### Element path
yt-icon-button[pressed="true"] > button
#### Snippet
`<button id="button" class="style-scope yt-icon-button" aria-pressed="true">`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Microsoft Edge version 92.0.902.67
This accessibility issue was found using Accessibility Insights for Web 2.31.1 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
